README.md

```diff
@@ -715,8 +715,20 @@ upcase("<sourceField>")
 
 Encodes a field value as URI. Aka percent-encoding.
 
+Options:
+
+- `allow_empty_values`: Sets whether to allow empty values in the filemap or to ignore these entries. (Default: `false`)
+- `compression`: Sets the compression of the file.
```

Comment:
These are not options for `uri_encode()`.

Reply:
cheeze - thx!